rest_framework_json_api/serializers.py

```diff
@@ -170,6 +170,47 @@ def get_field_names(self, declared_fields, info):
         fields = super(ModelSerializer, self).get_field_names(declared, info)
         return list(fields) + list(getattr(self.Meta, 'meta_fields', list()))
 
+    def to_representation(self, instance):
```

Comment:
Could you explain why this is overriding `to_representation` from the `ModelSerializer`? It seems like DJA is likely to diverge from the DRF upstream if this method is added. Is there some extra stuff added here that you could call out?

Reply:
@mblayman this is returning the JSON API identifier of the object without diving down the recursive rabbit whole. It looks like an elegant solution.

Reply:
Thanks for the second look, @aidanlister. I took a look at the DRF implementation and the method seems small enough that it would be hard for DJA to diverge too much from upstream.